.github/actions/prepare/action.yml

```diff
@@ -23,3 +23,4 @@ runs:
       run: |
         python manage.py makemigrations
         python manage.py migrate
+        python manage.py loaddata opportunityboard/fixtures/category_data.json
```

Comment:
Same point as was made in the other request. This should not be included in the GitHub actions workflows.